content_server.py

```diff
@@ -53,35 +73,11 @@ def process_item(self, identifier):
             if edition_identifier == identifier:
                 return identifier
         expect = identifier.urn
-        for message_identifier, status_message in messages.items():
```

Comment:
This code isn't necessary anymore because import_from_feed returns CoverageFailure objects if there's a problem.